text/0000-location.md

```diff
@@ -0,0 +1,440 @@
+
+### Defined GS1 Properties
+
+This Location RFC defines required and optional attributes (see below) for GS1
+physical locations such as manufacturing facilities, warehouses, distribution
+centers, retail stores, dock doors, etc. This implementation does not include
+attribution for specific industries, financial/tax account information, a 
+physical location extension, or digital locations and should be implemented 
+with supplemental RFC(s).
+
+**_REQUIRED FIELDS_**
+|GS1 Common Name|GS1 Attribute Name|Description|Example|Data Type|Min|Max|
+|---------------|------------------|-----------|-------|----|---|---|
+|Location Name 1|locationName|The name of the facility being described.|"Sunny Fresh Foods"|STRING|1|80|
+|Description|locationDescription|Free text, 178 characters.|"A Cargill production facility dedicated to serving high-quality egg products across various markets."|STRING| | |
+|Location Type|locationType|Multiple types allowed. All Suppliers: Org Entity, Order From, Remit To, Ship To. Healthcare Providers: Bill To, Deliver To, Order By, Order From, Org Entity, Paid By, Recall, Remit To, Ship From, Ship To.|"Ship From"|ENUM|7|48|
+|Address Line 1|addressLine1|The primary street address for your location. The USPS address is validated if Country = United States.| "206 W 4th Street"|STRING|1|80|
+|City|city|Name of the city of your location. The USPS address is validated if Country = United States.|"Monticello"|STRING|1|35|
+|State or Region|stateOrRegion|The state, province, or region using the standard two-letter abbreviation specified in ISO 3166-2:1998 country subdivision code [16].|"MN"|STRING|1|3|
+|Postal Code|postalCode|The ZIP or other postal code.|"55362-8524"|STRING|1|10|
+|Country|country|Country of your location. Spell out, do not use abbreviations.|"United States"|STRING|2|80|
+|Latitude, Longitude|latitude, longitude| |"44.986656, -93.258133"|LAT_LONG| | |
+|Contact Name|contactName| |"Jane Doe"|STRING| | |
+|Contact Email|contactEmail| |"jane_doe@example.com"|STRING| | |
+|Contact Phone|contactPhone|The location's primary phone number. Best practice is individual with assignment duty.|"937-435-3870"|STRING|1|30|
+|Create Date|createDate|Date this location becomes active.|"06/01/2015"|DATETIME| | |
+
+**_OPTIONAL FIELDS_**
+|GS1 Common Name|GS1 Attribute Name|Description|Example|Data Type|Min|Max|
+|---------------|------------------|-----------|-------|----|---|---|
+|Location Name 2|locationName2|A secondary facility name.|"Cargill Incorporated"|STRING|0|80|
+|Address Line 2|addressLine2|Any secondary information such as Suite, Floor, etc. The USPS address is validated if Country = United States.|"Suite 204"|STRING|0|80|
+|Address Line 3|addressLine3|Additional descriptive information that is not verified through the USPS data base. Best practice is to use AddressLine3 when there are multiple locations using the same USPS address. Examples: Billing office, cardiology lab, backroom, etc.| |STRING| | |
```

Comment:
Examples here would also be specific `dock door` or `airport terminal/gate`, yes?

Reply:
Yes, added both examples into the RFC